rust/arrow/src/compute/kernels/temporal.rs

```diff
@@ -17,34 +17,82 @@
 
 //! Defines temporal kernels for time and date related functions.
 
-use chrono::Timelike;
+use chrono::{Datelike, Timelike};
 
-use crate::array::*;
 use crate::datatypes::*;
 use crate::error::Result;
-
+use crate::{array::*, error::ArrowError};
```

Comment:
nit: rust-analyzer bunches imports in an off way (assuming you're using it). Sometimes they need reordering manually (e.g. grouping `Result` and `ArrowError`)

Reply:
Good catch. And you are right, I do use rust-analyzer.